Userland/Libraries/LibJS/Runtime/PropertyAttributes.h

```diff
@@ -58,6 +61,14 @@ class PropertyAttributes {
             m_bits &= ~Attribute::Configurable;
     }
 
+    void set_unimplemented(bool unimplemented = false)
```

Comment:
Hmm, this default parameter value feels wrong.
Saying `set_unimplemented()` would cause the property to be marked as NOT unimplemented!

Perhaps we don't need a default at all? :)

Reply:
I'm not sure what I was thinking there to be honest. I've removed this method entirely, since it wasn't actually being used.